srcpkgs/nheko/template

```diff
@@ -1,18 +1,26 @@
 # Template file for 'nheko'
 pkgname=nheko
-version=0.7.2
-revision=3
+version=0.8.2
+revision=1
 build_style=cmake
 hostmakedepends="qt5-host-tools qt5-qmake pkg-config qt5-declarative"
 makedepends="qt5-multimedia-devel qt5-svg-devel qt5-tools-devel fmt-devel
- tweeny spdlog mtxclient-devel boost-devel cmark-devel olm-devel json-c++
- libsodium-devel lmdb++ qt5-declarative-devel qt5-quickcontrols2-devel"
+ spdlog mtxclient-devel boost-devel cmark-devel olm-devel json-c++ lmdb++
+ qt5-declarative-devel qt5-quickcontrols2-devel qtkeychain-qt5-devel
+ gst-plugins-bad1-devel gst-plugins-base1-devel"
```

Comment:
```suggestion
 gst-plugins-bad1-devel gst-plugins-base1-devel xcb-util-wm-devel"
```
This will allow selecting a window during screen share instead of entire screen.

Reply:
Added